tests/NLog.UnitTests/LayoutRenderers/MessageTests.cs

```diff
@@ -219,5 +219,34 @@ public void MessageWithExceptionAndCustomSeparatorTest()
             logger.Debug(ex);
             AssertDebugLastMessage("debug", ex.ToString());
         }
+
+        [Fact]
+        public void SingleParameterException_OutputsSingleStackTrace()
```

Comment:
proposal: split the test for ${message} and ${exception}, then its better to read/maintain. 

(please create helpers for duplicate code)

Or use a parameterized test.

Reply:
I prefer the unit-test like this

Reply:
Ok. It's now anyway much better to read :)